src/main/java/org/fenixedu/academic/domain/raides/DegreeDesignation.java

```diff
@@ -35,18 +36,13 @@ public DegreeDesignation(String code, String description, DegreeClassification d
         setDegreeClassification(degreeClassification);
     }
 
-    public static DegreeDesignation readByNameAndSchoolLevel(String degreeDesignationName, SchoolLevelType schoolLevel) {
```

Comment:
wouldn't it be wiser to have an intermediate version with the old and new methods?